src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax;
+
+public class Ball {
+    String ballColor;
+    int ballNumber;
+
```

Comment:
Good idea to add a constructor with ballColor and ballNumber, to be able create a Ball in one line of code